.travis.yml

```diff
@@ -4,9 +4,9 @@ language: ruby
 matrix:
   include:
      - rvm: 2.0.0
-     - rvm: 2.1
-     - rvm: 2.2
-     - rvm: 2.3.0
+     - rvm: 2.1.10
```

Comment:
Yeah, they are supposed to but they don't (see https://travis-ci.org/bbatsov/rubocop/jobs/163619376#L129, for example).

I think it's because `rvm` chooses only between versions available, and versions like `2.2.5` don't seem available unless explicitly requested.